RUNBOOK.md

```diff
@@ -0,0 +1,88 @@
+## Overview 
+
+The Flow Playground is a web-based interactive IDE for running Cadence code.
+It also provides an environment for the [Cadence intro tutorials](https://docs.onflow.org/cadence/tutorial/01-first-steps). 
+
+The overall project consists of the web app (this) and an [API backend](https://github.com/onflow/flow-playground-api).
+
+The Playground Web App is implemented in React. The major components are as follows:
+
+### GraphQL / Apollo Client
+
+- All HTTP communication with the Playground API is done via `GraphQL` using the `Apollo` client. 
+- The GraphQL schema is defined by the Playground API in [schemal.graphql](https://github.com/onflow/flow-playground-api/blob/master/schema.graphql)
+  - This project uses Apollo Client's `localStorage` interface as well. 
+  - You can view the _local_ GraphQL schema in [local.graphql](src/api/apollo/local.graphql).
+- CRUD methods (wrapped Apollo client) are implemented in [projectMutator.ts](src/providers/Project/projectMutator.ts).
+- TypeScript typings and CRUD methods for Apollo are auto-generated using [GraphQL Code Generator](https://www.graphql-code-generator.com/).
+- After making changes to the `schema.local` you will need to run `npm run graphql:codegen` to auto-generate new typings and methods for Apollo Client.
+
+### Monaco Editor
+
+- The editor interface itself is implemented using [Monaco Editor](https://microsoft.github.io/monaco-editor/).
+- The editor component can be found here: https://github.com/onflow/flow-playground/tree/master/src/containers/Editor
+- The Cadence language definition (for linting and syntax highlighting) for Monaco can be found here: https://github.com/onflow/flow-playground/blob/master/src/util/cadence.ts
+
+### Cadence Language Server
+
+- The Cadence Language Server (protocol for Monaco) is implemented in Goland and compiled to WASM. 
```

Comment:
"(protocol for Monaco)" -- is this intended to explain language server?

Reply:
Yes, but adjusted the wording a bit